jpa/eclipselink.jpa.testapps/jpa.test.beanvalidation/src/main/java/org/eclipse/persistence/testing/models/jpa/beanvalidation/BeanValidationTableCreator.java

```diff
@@ -22,6 +22,7 @@ public BeanValidationTableCreator() {
         setName("BeanValidationEmployeeProject");
```

Comment:
Please update year in the copyright header into `Copyright (c) 2009, 2024 Oracle.....`.
It's about all updated files _*.java, persistence.xml_